Glog extraction

In the latest version, the downloaded zip (v0.3.4.zip) in my case does not extract into the `/glog_src/` folder during build. Currently using the catkin build system. When extracted and placed in the folder the system compiled file once I installed a gcc compiler using `sudo apt-get install build-essential`.

Original Error:
```
[glog_catkin] ==> '/catkin_ws/build/glog_catkin/build_env.sh /usr/bin/make --jobserver-fds=3,5 -j' in '/catkin_ws/build/glog_catkin'
[ 11%] Performing configure step for 'glog_src'
/bin/sh: 1: ./configure: not found
make[2]: *** [glog_src-prefix/src/glog_src-stamp/glog_src-configure] Error 127
make[1]: *** [CMakeFiles/glog_src.dir/all] Error 2
make: *** [all] Error 2
[glog_catkin] <== '/catkin_ws/build/glog_catkin/build_env.sh /usr/bin/make --jobserver-fds=3,5 -j' failed with return code '2'
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,14 +8,15 @@ include(ExternalProject)
 
 file(MAKE_DIRECTORY ${CATKIN_DEVEL_PREFIX}/include)
 
+set(VERSION 0.3.4)
 ExternalProject_Add(glog_src
-  DOWNLOAD_COMMAND wget --retry-connrefused --waitretry=1 --timeout=12 --tries 10 https://github.com/google/glog/archive/v0.3.4.zip
-  UPDATE_COMMAND ""
-  CONFIGURE_COMMAND cd ../glog_src && ./configure --with-pic 
+  DOWNLOAD_COMMAND wget --retry-connrefused --waitretry=1 --timeout=12 --tries 10 https://github.com/google/glog/archive/v${VERSION}.zip
+  UPDATE_COMMAND unzip -d . -o ../v${VERSION}.zip
+  CONFIGURE_COMMAND cd ../glog_src/glog-${VERSION} && ./configure --with-pic 
     --with-gflags=${gflags_catkin_PREFIX}
     --prefix=${CATKIN_DEVEL_PREFIX}
-  BUILD_COMMAND cd ../glog_src && make -j 8
-  INSTALL_COMMAND cd ../glog_src && make install -j 8
+  BUILD_COMMAND cd ../glog_src/glog-${VERSION} && make -j 8
+  INSTALL_COMMAND cd ../glog_src/glog-${VERSION} && make install -j 8
 )
 
 cs_add_library(${PROJECT_NAME} src/dependency_tracker.cc)

